src/libmeasurement_kit/ooni/templates.cpp

```diff
@@ -76,12 +79,26 @@ void http_request(Var<Entry> entry, Settings settings, http::Headers headers,
         settings["http/method"] = "GET";
     }
 
+    auto redact = [=](std::string s) {
+        if (!settings.get("save_real_probe_ip", false)) {
+            s = mk::ooni::scrub(s, probe_ip);
```

Comment:
Initially I was tipped off by the fact that you were calling this `probe_ip` in here and thought you were passing in `probe_ip` instead of `real_probe_ip`. Properly inspecting the diff I realised that is not the case, but I would advise changing this name to `real_probe_ip` to avoid confusion.